resolwe_bio/processes/expression/cuffnorm.yml

```diff
@@ -104,20 +104,23 @@
       experiment='{% for experiment in cuffquant %}{{ experiment.cxb.file }},{% endfor %}'
       replicates='{% for r in replicates %}{{r}},{% endfor %}'
       labels='{% for l in labels %}{{l}},{% endfor %}'
+      sample_names='{% for sample in cuffquant %}{{sample|sample_name}},{% endfor %}'
 
       labels=${labels%?}
```

Comment:
Labels will definitely not be an input any more. I will fix that. It's work in progress, forgot to label the pool request that way.